README.md

```diff
@@ -1,24 +1,82 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+## Recipe App
+This application keeps track of an user's transactions for different categories. It Allows a user to save categories, create transactions or delete categories, and Keeps track of the total transactions under a given category.
 
-* Ruby version
+## See live version
```

Comment:
- [ ] When I try to create the category, I got the following error. Kindly fix your live demo and make sure works properly and Make sure you run migration files in Heroku. You can look at [this video at min 6;00](https://www.youtube.com/watch?v=bQ2wXZf9irA)
>![image](https://user-images.githubusercontent.com/53402600/202658215-25f9e124-f9e9-43bc-a41d-441c480db13d.png)